functions/get_postgres_hosts.pp

```diff
@@ -0,0 +1,18 @@
+function rsan::get_postgres_hosts() {
+  if $settings::storeconfigs {
+    $postgres_hosts =
+                puppetdb_query('resources[certname] {
+                    type = "Class" and
+                    title = "pe_postgresql::server::install" and
+                    nodes {
+                      deactivated is null and
+                      expired is null
+                    }
+                  }').map |$data| { $data['certname'] }
+  } else {
```

Comment:
as there is no longer a conditional the exception of not having a postgres host returned needs handled differently

Reply:
Hey Marty, would this mean I need to implement some sort of error catch? So if puppetdb_query returns nothing/an error - make an exception to set 

`$postgres_hosts = []`